server/public/styles.css

```diff
@@ -67,16 +74,44 @@ header {
 }
 
 .single-entry-container {
-  margin-top: 15%;
+  padding-top: 15%;
   height: calc(100vh - 50px);
 }
 
+.single-entry-container th,
+.single-entry-container td {
+  padding-left: 10px;
+}
+
 .entry-information-table-th {
   width: 40%;
   vertical-align: top;
   text-align: start;
 }
 
+.show-menu {
+  width: 100%;
+  background-color: rgba(0, 0, 0, 0.3);
+  position: fixed;
+  height: calc(100vh - 50px);
+  bottom: 0;
+}
+
+.open-menu {
+  width: 40%;
```

Comment:
I recommend setting this to a hard `px` value. Usually applications have a a fixed width for their app drawers.